cpp/CMakeLists.txt

```diff
@@ -4,17 +4,9 @@ project(cpp)
 add_subdirectory(common)
 add_subdirectory(dataStructures)
 
-#For the shared library:
-set ( PROJECT_LINK_LIBS cppDS cppCommon)
-#link_directories( dataStructures/build )
-#link_directories( common/build )
+add_executable(testDS main.cpp)
 
-#Bring the headers, into the project
-include_directories(common/include)
```

Comment:
@Fraser999  main.cpp includes header files from common/include and dataStructures/include. If I don't put these lines, how will the inclusions be resolved ?